.templates/ideaPlugins/jsLinters/eslint.xml

```diff
@@ -2,6 +2,6 @@
 <project version="4">
   <component name="EslintConfiguration">
     <custom-configuration-file used="true" path="$PROJECT_DIR$/static-analysis/teammates-eslint.yml" />
-    <extra-options value="--ext .es6 --ignore-pattern '**/*.js'" />
+    <extra-options value="--ignore-pattern 'node_modules/**/*' 'src/main/webapp/js/*.js' 'src/main/webapp/test/*.js'" />
```

Comment:
Can you change this to `--ignore-pattern 'src/main/webapp/js/*.js' --ignore-pattern 'src/main/webapp/test/*.js' --ignore-pattern 'test-output/**/*.js'`?

Reply:
- I don't need to repeat `--ignore-pattern` three times, right?
- Alternatively, is there a way to specify the directory _to include_ instead of _to ignore_?

Reply:
> - I don't need to repeat `--ignore-pattern` three times, right?

According to the [ESLint documentation](https://eslint.org/docs/user-guide/command-line-interface#--ignore-pattern), the option has to repeated multiple times. I tried it without repeating the option and it didn't work.

- Alternatively, is there a way to specify the directory _to include_ instead of _to ignore_?

It doesn't seem like there is a way to do it because I think what IntelliJ intended is that we don't need to specify ignore patterns except for special cases, such files should have been automatically excluded if we configured our project correctly.

What I meant is that we can drop `Extra eslint options` altogether if we can configure the project properly. This means adding all the output folders to the list of excluded folders.

However, I guess this is not within the scope of this PR to address that's why I'm actually planning to mention it at the end of the PR as a future improvement.

There is, of course, the alternative way of using an ignore file which but since this is IntelliJ specific I did not want to commit an extra file.